README.md

```diff
@@ -34,20 +34,20 @@ The easiest way to manage multiple node versions in development is to use
 ## Get started
 
 * type `yarn` to install all dependencies
-* type `yarn dev:amo:dev` to start a local dev server that connects to a
-  hosted development environment
+* type `yarn amo:stage` to start a local server that connects to a
+  hosted staging server
 
 ## Development commands
 
 Here are some commands you can run:
 
 | Command                     | Description |
 |-----------------------------|-------------|
-| yarn dev:amo                | Start the dev server/proxy (for amo) using data from Docker |
-| yarn dev:amo:dev            | Start the dev server/proxy (for amo) using data from the dev server (https://addons-dev.allizom.org/) |
-| yarn dev:amo:no-proxy       | Start the dev server without a proxy (for amo) using data from Docker |
-| yarn dev:amo:stage          | Start the dev server/proxy (for amo) using data from the staging server (https://addons.allizom.org/) |
-| yarn dev:disco              | Start the dev server (for Discovery Pane) using data from Docker |
+| yarn amo                    | Start the dev server/proxy (for amo) using data from Docker |
+| yarn amo:dev                | Start the dev server/proxy (for amo) using data from the dev server (https://addons-dev.allizom.org/) |
+| yarn amo:no-proxy           | Start the dev server without a proxy (for amo) using data from Docker |
+| yarn amo:stage              | Start the dev server/proxy (for amo) using data from the staging server (https://addons.allizom.org/) |
+| yarn disco                  | Start the dev server (for Discovery Pane) using data from the dev server (https://addons-dev.allizom.org/) |
```

Comment:
I feel like we should change that, but it can be a separate patch. I guess Disco data is pretty standard though.